lib/cache.js

```diff
@@ -88,7 +88,7 @@ const saveConfigToManifest = async (config, fetchedModules) => {
  */
 const fetchScript = async (script, proxyAgent) => {
   try {
-    log(4, `[cache] Fetching script ${script}.`);
+    log(4, `[cache] Fetching script - ${script}.js`);
```

Comment:
Adding the `.js` extension here will result in a doubled `.js` in case of custom scripts (like moment). To resolve this, we can simply place the log below the if with the `script.endsWith('.js')` condition.

Reply:
You're right, fixed ✅